### ISSUE TITLE
Mailer content confuses roles

### ISSUE THREAD

**Issue by F3PiX** [rails-girls-summer-of-code/rgsoc-teams#476]
When I assign myself as a Supervisor, a confirmation mail is sent by the mailer. In the body of the mail it says correctly that I am a Supervisor, but in the second part it adresses me as a coach.
This part of the mail is supposed to show only if role == "coach".
I don't see why it is showing up with the supervisor role as well. Anybody? Could it be misplaced slim syntax?

Mail: "You have been added to the 6-melting as a _supervisor_.

Visit the team at http://localhost:3000/teams/37
Please confirm that you are indeed willing to join the team as their _coach_. You can do so on the team's overview page listed above (make sure you are logged in using your Github account)."

This is the conditional:
https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/views/role_mailer/user_added_to_team.html.slim#L16
in this file:
rgsoc-teams/app/views/role_mailer/user_added_to_team.html.slim

**Comment by carpodaster** [comment]
Hello @F3PiX, thanks for reporting this. The email I got had the same error. I just checked the code and couldn't find any apparent errors. I also cannot reproduce it in my local dev setup.

My initial idea was what the roles have mistakingly been added as coach-roles first (which would trigger the after create callback) and then later edited to supervisor or mentor. That's impossible, though, as the mail body correctly states that one has been added as a supervisor/mentor.
